CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to sku
+
+⚠️ 🌏 👀 First and foremost, remember that this repo is **open source**.
+
+Don't post anything or commit any code that isn't ready for the entire world to see. Avoid making specific reference to internal processes or features under development. While a lot of this information is probably harmless, it's better to be safe than sorry.
+
+If you work at SEEK and run into issues along the way, or even if you find some of these steps confusing or intimidating, please reach out to your friendly neighbourhood developer in the sku development Slack channel. We'll be super excited to help out!
+
+## Setup
+
+First, install [Node.js](https://nodejs.org/).
+
+After cloning the project, install the dependencies:
+
+```bash
+$ npm install
+```
+
+## Workflow
+
+Before starting your work, first ensure you're in the `master` branch and that you've pulled down the latest changes:
+
+```bash
+$ git checkout master
+$ git pull
+```
+
+Next, create a new branch for your work, with an appropriate name for your change:
+
+```bash
+$ git checkout -b add-my-cool-new-feature
+```
+
+To run the test suite locally:
+
+```bash
+$ npm test
+```
+
+Note that the test suite needs to pass for your changes to be accepted, so it's worth running this locally during development and before committing.
+
+Once you've made the desired changes and you're ready to commit, first stage your local changes:
+
+```bash
+$ git add .
```

Comment:
Yeah, possibly—this was adapted from the style guide, which was written so that designers with less technical experience could follow along, even if we have to help tidy a few things up. This project is for a much more savvy audience, so we can probably trim out a lot of this stuff.